app/controllers/kaui/credits_controller.rb

```diff
@@ -23,13 +23,13 @@ def create
     # No need to show the newly created invoice for account level credits
     should_redirect_to_invoice = credit.invoice_id.present?
 
-    credit = credit.create(true, current_user.kb_username, params[:reason], params[:comment], options_for_klient)
+    credits = credit.create(true, current_user.kb_username, params[:reason], params[:comment], options_for_klient)
     flash[:notice] = 'Credit was successfully created'
 
     if should_redirect_to_invoice
-      redirect_to kaui_engine.account_invoice_path(credit.account_id, credit.invoice_id)
+      redirect_to kaui_engine.account_invoice_path(credits[0].account_id, credits[0].invoice_id)
     else
-      redirect_to kaui_engine.account_path(credit.account_id)
+      redirect_to kaui_engine.account_path(credits[0].account_id)
```

Comment:
FYI (but I don't think it's worth fixing right now) we're not quite consistent in the Ruby client. The tax API will return the refreshed credit instead: https://github.com/killbill/killbill-client-ruby/blob/8c59e5f3349f9caafccf03257125c5ca8a0e3b63/lib/killbill_client/models/invoice_item.rb#L98.